Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -824,6 +848,8 @@ public async Task GetMultipleBlocksAtOnce()
             {
                 await swarmA.StopAsync();
                 await swarmB.StopAsync();
+
+                swarmB.Dispose();
```

Comment:
```suggestion
                swarmA.Dispose();
                swarmB.Dispose();
```

Call `Dispose` is not required? or missed?

Reply:
`swarmA.Dispose()` isn't required because `swarmA` is `_swarms[0]` and `_swarms[0]` will be dispose in `SwarmTest.Dispose()` (it's quite tricky 😕 )